catalog/README.md

```diff
@@ -1,39 +1,26 @@
-<!--
-This README describes the package. If you publish this package to pub.dev,
-this README's contents appear on the landing page for your package.
+# Catalog' Flutter template
 
-For information about how to write a good package README, see the guide for
-[writing package pages](https://dart.dev/guides/libraries/writing-package-pages).
-
-For general information about developing packages, see the Dart guide for
-[creating packages](https://dart.dev/guides/libraries/create-library-packages)
-and the Flutter guide for
-[developing packages and plugins](https://flutter.dev/developing-packages).
--->
-
-TODO: Put a short description of the package here that helps potential users
-know whether this package might be useful for them.
+Catalog is a Flutter package that provides a base implementation for UX/UI components based on the design template of Xmartlabs.
+It aims to simplify and expedite the development process for Flutter projects, ensuring a consistent and visually appealing user interface.
 
 ## Features
```

Comment:
Maybe we can add something about the theme here

Reply:
Also we can explain where each part is, kind of what we have in the template readme with the folder structure.
So it's easier to modify later on